app/views/_navbar.erb

```diff
@@ -5,28 +5,48 @@
     </section>
 
     <section>
+
+      <% unless current_path == '/spaces' %>
+      <form action='/spaces' method="get">
+          <button type="submit">Available spaces</button>
+      </form>
+      <%end%>
+
+      <% if current_path == '/spaces' || current_path == '/requests' %>
       <form action='/spaces/new' method="get">
           <button type="submit">Create space</button>
       </form>
+      <%end%>
+
+      <% unless current_path == '/requests' %>
       <form action='/requests'>
         <button type="submit">Check requests</button>
       </form>
+      <%end%>
+
       <form action='/session' method="post">
         <input type='hidden' name='_method' value='delete'>
         <button type="submit">Sign out</button>
       </form>
+
   <%else%>
   <section class="navbar-welcome">
     Makers BnB
   </section>
     <section class="navbar-nav">
-     <form action='/session/new'>
-       <button type="submit">Log in</button>
-     </form>
 
-     <form action='/user/new'>
-       <button type="submit">Register</button>
-     </form>
+      <% unless current_path == '/session/new' %>
+        <form action='/session/new'>
+        <button type="submit">Log in</button>
+        </form>
+      <%end%>
+
+      <% unless current_path == '/user/new' %>
```

Comment:
I think we should remove the **Register** button in favor of a sign up page on the homepage as the spec requests.